src/main/java/com/minecolonies/coremod/blocks/AbstractBlockHut.java

```diff
@@ -202,7 +202,7 @@ public boolean onBlockActivated(
 
             if (building != null
                   && building.getColony() != null
-                  && building.getColony().getPermissions().hasPermission(playerIn, Permissions.Action.ACCESS_HUTS))
+                  && building.getColony().getPermissions().hasPermission(playerIn, Action.ACCESS_HUTS))
```

Comment:
I think this belong to a different pr, and could be merge well before dman 2.0 is ready